Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -8,17 +8,23 @@
 
 
 // Controller
-#import "WMFArticleFetcher.h"
 #import "WMFArticleViewController.h"
 #import "WebViewController.h"
 #import "UIViewController+WMFStoryboardUtilities.h"
 #import "WMFSaveButtonController.h"
 #import "WMFPreviewController.h"
 #import "WMFArticleContainerViewController_Transitioning.h"
+#import "WMFArticleHeaderImageGalleryViewController.h"
```

Comment:
Yeah, I thought of that too. There's a big legacy code cleanup coming which might help with this (removing a lot of leftover `WMFArticleViewController` stuff), but it would be nice to break out responsibilities a bit more.